workflows/pipe-common/scripts/mount_storage.py

```diff
@@ -246,6 +246,14 @@ def init_tmp_dir(tmp_dir, task_name):
         if StorageMounter.create_directory(fuse_tmp, task_name):
             AzureMounter.fuse_tmp = fuse_tmp
 
+    def mount(self, mount_root, task_name):
+        super(AzureMounter, self).mount(mount_root, task_name)
+        # add resolved ip address for azure blob service to /etc/hosts (only once per account_name)
+        account_name, _, _ = self._get_credentials(self.storage)
+        command = "grep {account_name} /etc/hosts || getent hosts {account_name}.blob.core.windows.net ".format(account_name=account_name) \
```

Comment:
What if ip address changes since last addition to the /etc/hosts? 
E.g.:
1. We ran some tool
2. IP was added to /etc/hosts
3. We paused/resumed a run
4. IP is changed, but we still have the old one in /etc/hosts

Is it handled?

Reply:
No it's not, we can do replacement here instead of current solution.